pom.xml

```diff
@@ -55,7 +55,7 @@
         <!-- plugin dependencies -->
 
         <version.quarkus>3.2.7.Final</version.quarkus>
-        <version.rabbit.amqp-client>5.16.1</version.rabbit.amqp-client>
+        <version.rabbit.amqp-client>5.18.0</version.rabbit.amqp-client>
```

Comment:
this is not compatible with the version used in quarkus- rabbitmq messaging there is good chance to clash with vert.x

Reply:
I bumped this because of: https://github.com/iris-events/iris/security/dependabot/7

Reply:
Reverted to 5.15.1